docs/usage_and_configuration/the_basics.md

```diff
@@ -259,10 +259,14 @@ expressions by Black. Use `[ ]` to denote a significant space character.
 line-length = 88
 target-version = ['py37']
 include = '\.pyi?$'
+# 'extend-exclude' excludes files or directories in addition to the defaults
 extend-exclude = '''
 # A regex preceded with ^/ will apply only to files and directories
 # in the root of the project.
-^/foo.py  # exclude a file named foo.py in the root of the project (in addition to the defaults)
+(
+  ^/foo.py    # exclude a file named foo.py in the root of the project
+  | *.py.bak  # exclude a file with '.py.bak' suffix within the project
```

Comment:
Does black by default try to format these? If so, should we exclude this?

Reply:
In this example, `'\.pyi?$'` is specified and `*.py.bak` is excluded by default.
So in 9a6f541 I changed this example to `*_pb2.py` which is a generated stub file by Protocol Buffer.